community/cypher/frontend-3.3/src/main/scala/org/neo4j/cypher/internal/frontend/v3_3/ast/Clause.scala

```diff
@@ -86,15 +86,46 @@ case class EmitGraph(graphName: Expression)(val position: InputPosition) extends
 case class Start(items: Seq[StartItem], where: Option[Where])(val position: InputPosition) extends Clause {
   override def name = "START"
 
-  override def semanticCheck =
-    SemanticError("START is no longer supported. You can accomplish the same thing using MATCH and WHERE", position)
+  override def semanticCheck: SemanticCheck = (state: SemanticState) => {
+
+    val query = rewrittenQuery
+    val newState = state.addNotification(DeprecatedStartNotification(position, query))
+    SemanticCheckResult(newState, Seq(SemanticError(
+      s"""START is deprecated, use: `$query` instead.
+       """.stripMargin, position)))
+  }
+
+  private def rewrittenQuery: String = {
+    val rewritten = items.map {
+      case NodeByIdentifiedIndex(variable, index, key, expression) =>
+        s"CALL db.nodeLegacyIndexSeek('$index', '$key', '${expression.asCanonicalStringVal}') YIELD node AS ${variable.asCanonicalStringVal}"
```

Comment:
This should talk about manual indexes and not legacy stuff, right? Other examples in the next couple of lines